interfaces/mount/entry.go

```diff
@@ -168,3 +169,24 @@ func LoadFSTab(reader io.Reader) ([]Entry, error) {
 	}
 	return entries, nil
 }
+
+// SaveFSTab writes a list of entries to a fstab-like file.
+//
+// The supported format is described by fstab(5).
+//
+// Note that there is no support for comments, both the LoadFSTab function and
+// SaveFSTab just ignore them.
+//
+// Note that there is no attempt to use atomic file write/rename tricks. The
```

Comment:
Yeah, those discussions made me rethink this a little. Please have a look at #3135